packages/url_launcher/url_launcher_web/pubspec.yaml

```
@@ -2,11 +2,11 @@ name: url_launcher_web
 description: Web platform implementation of url_launcher
 repository: https://github.com/flutter/plugins/tree/main/packages/url_launcher/url_launcher_web
 issue_tracker: https://github.com/flutter/flutter/issues?q=is%3Aissue+is%3Aopen+label%3A%22p%3A+url_launcher%22
-version: 2.0.7
+version: 2.0.8
 
 environment:
   sdk: ">=2.12.0 <3.0.0"
-  flutter: ">=2.0.0"
+  flutter: ">=2.10.0"
```

```suggestion
  flutter: ">=2.10.0-0"
```

I suspect this needs to be `>=2.10.0-0` so it picks up dev versions too, otherwise this won't work until flutter `master` goes to `2.11.0-X.pre`, IIRC?

---

I didn't do this since I thought pub would give a warning, which would fail CI. Maybe it only does that for `sdk` and not for `flutter`.